backend/app/routers/feeds.py

```diff
@@ -50,9 +50,13 @@ async def check_feed_listeners(
         # Verify whether resource_id is found when searching the specified criteria
         feed_match = check_search_result(es_client, file_out, feed.search)
         if feed_match:
-            for listener in feed.listeners:
-                if listener.automatic:
-                    listener_ids_found.append(listener.listener_id)
+            feed_criteria = feed.search.criteria
+            for criterea in feed_criteria:
+                if criterea.field == 'content_type_main' and criterea.value == file_out.content_type.main_type:
+                    for listener in feed.listeners:
+                        # TODO not sure if this check is still needed
+                        if listener.automatic:
```

Comment:
Just curious, who's writing this "automatic" flag? Is it a part of the listener definition? If so could someone show an example?

Reply:
it can be set when registering the extractor, or changed later. clowder v1 extractor_info will set this true by default.